src/index.d.ts

```diff
@@ -14,6 +14,30 @@ declare module 'bootstrap-vue' {
 
   export default VuePlugin
 
+  type VueVariant = 'active' | 'success' | 'info' | 'warning' | 'danger'
```

Comment:
There are actually quite a few more variants, depending on where they are used, such as `primary`, `secondary`, `light` and `dark`... and people can defined their own variants in CSS/SCSS as well.

See the Bootstrap Standard variants at: https://bootstrap-vue.js.org/docs/reference/color-variants

So I am not sure if restricting the values the the above 5 types is goo.. (as well, `active` is not an official variant, as it is a class unto it's own)

Reply:
According to the docs for `variant`, here: https://bootstrap-vue.js.org/docs/components/table#field-definition-reference

> Apply contextual class to all the `<th>` and `<td>` in the column - `active`, `success`, `info`, `warning`, `danger`. These variants map to classes thead-${variant} (in the header), table-${variant} (in the body), or bg-${variant} (when table prop dark is set).

So, is the documentation here incorrect? If that's the case, we should update the docs as well (which I'm willing to do in a further PR).

Reply:
Oh, I just read your comment below... Perhaps I'll rename this to `TableVariant`, and define `variant` as `variant: TableVariant | string`. Would that solve our issue?

Reply:
Ah, right `table-active` is an actual variant.  The variant reference docs should be updated to include the extra `active` variant.  And maybe the table docs should have a link back to the variant docs.

Reply:
and `variant: TableVariant | string` would be good.